src/CodingMilitia.PlayBall.GroupManagement.Web/IoC/ServiceCollectionExtensions.cs

```diff
@@ -42,5 +44,10 @@ public static TConfig ConfigurePOCO<TConfig>(this IServiceCollection services, I
             services.AddSingleton(config);
             return config;
         }
+
+        public static IServiceCollection AddDbInitializer<T> (this IServiceCollection services) where T : DbContext

```

Comment:
This method should be added to the extracted StartupTasks project, so it can be reused in other web projects.

Reply:
will do